scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -64,6 +77,13 @@
                           (total-resource node-name->consumed :cpus))
     (monitor/set-counter! (metrics/counter "consumption-mem" compute-cluster-name)
                           (total-resource node-name->consumed :mem))
+    (run!
```

Comment:
Why not:
(doseq [gpu-mode gpu-models]
  ....)

Reply:
@scrosby What do you have against using `run!` here?

Reply:
done